fritzconnection/lib/fritzwlan.py

```diff
@@ -189,17 +235,22 @@ def beacontype(self):
         settings and `None, 11i, 11iandWPA3, OWETrans` for the guest
         network.
         """
-        return self.get_info()['NewBeaconType']
+        return self.get_info()["NewBeaconType"]
 
     @property
     def channel(self):
         """The WLAN channel in use"""
-        return self.channel_info()['NewChannel']
+        return self.channel_info()["NewChannel"]
 
     @property
     def alternative_channels(self):
-        """Alternative channels (as string)"""
-        return self.channel_info()['NewPossibleChannels']
+        """Alternative channels (as string)."""
+        return self.channel_info()["NewPossibleChannels"]
+
+    @property
+    def get_wifi_qr_code(self, kind):
+        """Returns a file-like object providing a bytestring representing a qr-code for WLAN access."""
+        return get_wifi_qr_code(self, kind)
```

Comment:
The problem is that Hasssio's integration uses this method but can't find it:

https://github.com/home-assistant/core/blob/b20a286b5bbc70e48565f8bc390e1b20b37e07e6/homeassistant/components/fritz/image.py#L66-L68

Reply:
It is called and processed, but is not available in the IDE

Reply:
Do I understand this right: it works fine, but the IDE is unable to list `get_wifi_qr_code()` as a the method of a `FritzGuestWLAN` class? 

In this case that can be because of some meta-programming in the module; IDEs are unable to resolve code changes at import time by just making a static code check.

Reply:
Ok thanks for your explanation I removed the property